python/cuspatial/cuspatial/core/geodataframe.py

```diff
@@ -126,50 +135,101 @@ def _copy_type_metadata(
 
         return type_copied
 
+    def _init_from_dict_like(self, data, index):
```

Comment:
It looks like this function isn't used?

Reply:
Ah perhaps I avoided implementing it after all.